src/PresentationalComponents/RuleDetails/RuleDetails.js

```diff
@@ -0,0 +1,145 @@
+import React, { Component } from 'react';
+import { Ansible, Battery, Shield, routerParams } from '@red-hat-insights/insights-frontend-components';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Grid, GridItem } from '@patternfly/react-core';
+import { addNotification } from '@red-hat-insights/insights-frontend-components/components/Notifications';
+import ReactMarkdown from 'react-markdown/with-html';
+import * as AppActions from '../../AppActions';
+import { SYSTEM_TYPES, RISK_OF_CHANGE_LABEL, RISK_OF_CHANGE_DESC, TOTAL_RISK_LABEL, TOTAL_RISK_DESC } from '../../AppConstants';
```

Comment:
Checkout line eight, when your brining in a bunch of exports, might as well bring em all in under an alias

Reply:
:+1: